include/glow/Graph/Graph.h

```diff
@@ -268,6 +268,15 @@ class Function final : public Named {
   FullyConnectedNode *createFullyConnected(llvm::StringRef name,
                                            NodeValue input, size_t outDepth);
 
+  /// Create a row-wise fully connected node. This node is only used in
+  /// quantization. Parm \p input and \p B are quantized in regular way,
+  /// \p W is the constant weights and will be row-wise quantized during
+  /// compiler time.
+  RowwiseQuantizedFullyConnectedNode *createRowwiseQuantizedFullyConnected(llvm::StringRef name,
+                                                         NodeValue input,
+                                                         Node *W, Node *B,
```

Comment:
Why do we accept `Node*` here, and not `Storage` or `Variable`?

Reply:
Since previously, I added one node to do the row-wise quantization, and then W was the result of that RowWiseQuantization node. I forgot to update this interface... Now it is updated to "Variable".